eZ/Publish/Core/MVC/Symfony/Routing/Generator/UrlAliasGenerator.php

```diff
@@ -55,6 +55,19 @@ class UrlAliasGenerator extends Generator
      */
     private $configResolver;
 
+    /**
+     * Array of characters that are potentially unsafe for output for (x)html, json, etc,
+     * and respective url-encoded value
+     *
+     * @var array
+     */
+    protected $unsafeCharMap = array(
+        '"' => '%22',
+        "'" => '%27',
+        '<' => '%3C',
+        '>' => '%3E',
+    );
```

Comment:
I'd prefer have this in a container parameter, injected in this service. This would ease further changes / tweaks